src/enums/index.ts

```diff
@@ -1,138 +1,153 @@
-import { makeClass } from "./../utils/make-class";
-const assert = require("assert");
-const _ = require("lodash");
-const { parseBytes, serializeUIntN } = require("./../utils/bytes-utils");
-const enums = require("./definitions.json");
-
-function transformWith(func, obj) {
-  return _.transform(obj, func);
+import { serializeUIntN } from "./../utils/bytes-utils";
+import * as enums from "./definitions.json";
+
+const TYPE_WIDTH = 2;
+const LEDGER_ENTRY_WIDTH = 2;
+const TXN_TYPE_WIDTH = 2;
```

Comment:
Sometimes you write "TXN" and other times you use "TRANSACTION". Can we make these consistent? 

I prefer that you don't use abbreviations (because the definition of 'common abbreviations' often differ between individual devs or teams) but I don't care which direction you go, as long as we're consistent.